src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/timeseries/ComputedAnomalyKmeansModelParameter.scala

```diff
@@ -0,0 +1,111 @@
+package com.microsoft.partnercatalyst.fortis.spark.analyzer.timeseries
+
+import java.time.{ZoneId, ZonedDateTime}
+
+import com.microsoft.partnercatalyst.fortis.spark.dto.TopicCount
+
+case class ComputedAnomalyKmeansModelParameter(
+  key: String,
+  k: Int,
+  maxIter: Int,
+  periodFeature: TopicCount => Double
+) extends Serializable
+
+object ComputedAnomalyKmeansModelParameter {
+
+  val parametersForAnalysis: Set[ComputedAnomalyKmeansModelParameter] = PeriodType.all.flatMap(kmeansParameters(_, 1, 1))
+
+  val parametersForAnalysisByKey: Map[String, ComputedAnomalyKmeansModelParameter] = parametersForAnalysis.groupBy(_.key).mapValues(v=>v.head)
+
+  def kmeansParameters(periodType: PeriodType,
+                       uniqueSourcesCount: Int,
+                       uniqueTopicsCount: Int): Seq[ComputedAnomalyKmeansModelParameter] = {
+    periodType match {
+      case PeriodType.Minute => Seq(
+        ComputedAnomalyKmeansModelParameter(
+          ComputedAnomalyKmeansModelKey.Minute,
+          60 * uniqueSourcesCount * uniqueTopicsCount,
+          20,
```

Comment:
Maybe you could make the k for kmeans a constant to make it a little more clear?